src/Cake.Core.Tests/Fixtures/ScriptHostFixture.cs

```diff
@@ -18,9 +19,9 @@ public TestingScriptHost(ICakeEngine engine, ICakeContext context)
             {
             }
 
-            public override CakeReport RunTarget(string target)
+            public override Task<CakeReport> RunTarget(string target)
             {
-                return new CakeReport();
+                return System.Threading.Tasks.Task.FromResult(new CakeReport());
```

Comment:
Ahh, that makes sense :). And it does look better than `using Task = System.Threading.Tasks.Task`; since it's only used one.